stable/bookstack/templates/deployment.yaml

```diff
@@ -89,7 +89,7 @@ spec:
           - name: uploads
             mountPath: /var/www/bookstack/public/uploads
           - name: storage
-            mountPath: /var/www/bookstack/public/storage
```

Comment:
I'm guessing the current public/storage never gets used, and attachments are written in the ephemeral path storage/uploads. Causing files to disappear when you restart the container?

Reply:
Exactly. 
I haven't seen anything saved under public/storage at all. I have seen attachments disappearing though - that's also mentioned in the issue. 
The `public/storage` is not mentioned in the official bookstack docs on backup section- I presume is a bug or an old version related path, although haven't had any confirmation from the Dockerfile repo owner.

Reply:
![image](https://user-images.githubusercontent.com/2114993/55412339-87368380-5567-11e9-9182-6452bb0d30e8.png)

Reply:
Your change makes the chart adhere to these directories, which is a good thing. ;-)

Reply:
Which is all listed in your description, my bad, i jumped straight to the diffs!